addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshOptionsPanelTab.java

```diff
@@ -161,9 +169,35 @@ private void newPayloadButtonAction() {
         }
     }
 
+    private void addChangeListener(
+            JTextField textField, BiConsumer<JTextField, DocumentEvent> action) {
+        textField
+                .getDocument()
+                .addDocumentListener(
+                        new DocumentListener() {
+
+                            @Override
+                            public void changedUpdate(DocumentEvent e) {
+                                action.accept(textField, e);
```

Comment:
Why not call `refreshPayloadButtonEnabledState` directly? The `BiConsumer` does not seem to be needed, the arguments are not used, if you still prefer to use a lambda consider changing to `Runnable` and use method reference.